tutorials/blockchain-analyze/analyze-blockchain-query.md

```diff
@@ -158,20 +158,20 @@ fees to decrease.
      tx_count as "tx volume",
      average(stats_fee_sat) as fees
     FROM one_hour_transactions
-    WHERE bucket > NOW() - INTERVAL '2 days'
+    WHERE bucket > date_add('2023-11-22 00:00:00+00', INTERVAL '-2 days')
```

Comment:
Hi @combscCode, where did you find this function:

```
 select date_add('2023-11-22 00:00:00+00', INTERVAL '-2 days');
ERROR:  function date_add(unknown, interval) does not exist
LINE 1: select date_add('2023-11-22 00:00:00+00', INTERVAL '-2 days'...
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
tsdb=>
tsdb=> \df date_add
                       List of functions
 Schema | Name | Result data type | Argument data types | Type
--------+------+------------------+---------------------+------
(0 rows)
```

In this case, we could just use plain dates. However, I'd still recommend fixing the dataset updates rather than making this change.

Reply:
huh, I found it here! https://www.postgresql.org/docs/current/functions-datetime.html

Yeah, fix the issue however you prefer! Feel free to close this PR if y'all are gonna fix the dataset 👍

Reply:
Sorry, in my reference, now I checked I was connecting to a Postgresql 13 version 🤦